cpp/include/raft/spatial/knn/detail/ann_kmeans_balanced.cuh

```diff
@@ -303,8 +307,8 @@ void calc_centers_and_sizes(const handle_t& handle,
     centers,
     centers,
     sizes_f,
-    (int64_t)dim,
-    (int64_t)n_clusters,
+    static_cast<int64_t>(dim),
```

Comment:
Thanks so much for doing this!  It tends to matter more for pointer conversions, but integer conversions can also be a source of bugs, so it's nice to be able to search for `static_cast`.

Reply:
Yes, no problem. I don't always use `static_cast` for trivial conversions but I don't mind using it.